scripts/rtma3d/exrtma3d_gsianl.ksh

```diff
@@ -415,8 +438,8 @@ done
 #${MV} ${SFCOBS_USELIST}/current_mesonet_uselist.txt ${SFCOBS_USELIST}/${PDYm1}_mesonet_uselist.txt
 #scpreq Edward.Colon@dtn-jet.boulder.rdhpcs.noaa.gov:/mnt/lfs4/HFIP/hfv3gfs/Edward.Colon/reject_use_lists/current_mesonet_uselist.txt ${SFCOBS_USELIST}/
 #fi
-${CP} ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
-${CP} ${SFCOBS_USELIST}/current_mesonet_uselist.txt gsd_sfcobs_uselist.txt
+#${CP} ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
```

Comment:
same question as I raised in exakrtma3d_gsianl.ksh.
But it is OK to keep this modification. 
We can make it more flexible later.

Reply:
still just a suggestion. It is good for this PR.

Reply:
@GangZhao-NOAA OK, see comment in exakrtma3d_gsianl.ksh for my suggestion on how to make this more flexible.